cap-ci/config.yaml

```diff
@@ -10,6 +10,7 @@ jobs:
   sync-integration-tests: true
   upgrade: true
   stratos: true
+  clean-kubecf: true
```

Comment:
clean up should not be an option in my opinion, but its ok. This card is not just about kubecf cleanup. This is about all installation cleanup including stratos and stratos metrics.

Reply:
added `stratos` and `metrics` cleanup.

Reply:
you will have to rename this flag if you are cleaning up stratos and metrics as well with this

Reply:
DONE.